SU2_CFD/src/solvers/CEulerSolver.cpp

```diff
@@ -6614,16 +6618,20 @@ void CEulerSolver::BC_Inlet(CGeometry *geometry, CSolver **solver_container,
           V_inlet[nDim+3] = Energy + Pressure/Density;
 
           break;
+        }
+        /*--- Mass flow has been specified at the inlet. ---*/
 
-          /*--- Mass flow has been specified at the inlet. ---*/
-
-        case INLET_TYPE::MASS_FLOW:
+        case INLET_TYPE::MASS_FLOW: {
```

Comment:
What is the intention behind wrapping each `case` in curly braces? I can only think of some variable-scope implications, but I guess the `switch` already introduces that variable scope "barrier" :thinking:

Reply:
It's the only way to introduce local variables inside switch cases.